services/api-server/tests/unit/pact_broker/test_pact_checkout_release.py

```diff
@@ -0,0 +1,144 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+import os
+
+import pytest
+from fastapi import FastAPI
+from models_library.api_schemas_webserver.licensed_items_checkouts import (
+    LicensedItemCheckoutRpcGet,
+)
+from pact.v3 import Verifier
+from pytest_mock import MockerFixture
+from simcore_service_api_server._meta import API_VERSION
+from simcore_service_api_server.api.dependencies.authentication import (
+    Identity,
+)
+from simcore_service_api_server.api.dependencies.resource_usage_tracker_rpc import (
+    get_resource_usage_tracker_client,
+)
+from simcore_service_api_server.api.dependencies.webserver_rpc import (
+    get_wb_api_rpc_client,
+)
+from simcore_service_api_server.services_rpc.resource_usage_tracker import (
+    ResourceUsageTrackerClient,
+)
+from simcore_service_api_server.services_rpc.wb_api_server import WbApiRpcClient
+
+
+def mock_get_current_identity() -> Identity:
```

Comment:
I wonder why dont you just create a constant for this and use it everywhere or a fxiture in `conftest.py`